lib/util/player_configuration.js

```diff
@@ -75,6 +75,7 @@ shaka.util.PlayerConfiguration = class {
       updateExpirationTime: 1,
       preferredKeySystems: [],
       keySystemsMapping: {},
+      parseInbandPsshEnabled: false,
```

Comment:
I think the default value should be true for Xbox.

Reply:
Enabling this by default makes me nervous. I would prefer to leave it false until we have production data showing that it has no adverse effects.

Reply:
Do you have new data about it?

Reply:
We will not have production data on this for a while. I hoped to complete code review process before deploying. If you all want this enabled by default for Xbox, I'm ok with that.

Reply:
@joeyparrish what do you think?

Reply:
I think it's best to have platform-specific defaults in cases like this.  It may be difficult for service providers to discover that they need this flag enabled on Xbox.  So long as it doesn't hurt anything for other kinds of content, let's make the default true for Xbox.